src/pages/workspace/taxes/WorkspaceEditTaxPage.tsx

```diff
@@ -0,0 +1,156 @@
+    const canEdit = policy && canEditTaxRate(policy, taxID);
+
+    const toggleTaxRate = () => {
+        if (!currentTaxRate) {
+            return;
+        }
+        setPolicyTaxesEnabled(policyID, [taxID], !!currentTaxRate.isDisabled);
+    };
+
+    const deleteTaxRate = () => {
+        if (!policyID) {
+            return;
+        }
+        deletePolicyTaxes(policyID, [taxID]);
+        setIsDeleteModalVisible(false);
+        Navigation.goBack();
+    };
+
+    const threeDotsMenuItems: ThreeDotsMenuItem[] = useMemo(
+        () => [
+            {
+                icon: Expensicons.Trashcan,
+                text: translate('common.delete'),
+                onSelected: () => setIsDeleteModalVisible(true),
+            },
+        ],
+        [translate],
+    );
+
+    if (!currentTaxRate) {
+        return <NotFoundPage />;
+    }
+
+    return (
+        <AdminPolicyAccessOrNotFoundWrapper policyID={policyID}>
+            <PaidPolicyAccessOrNotFoundWrapper policyID={policyID}>
+                <ScreenWrapper
+                    testID={WorkspaceEditTaxPage.displayName}
+                    style={styles.mb5}
+                >
+                    <View style={[styles.h100, styles.flex1]}>
+                        <HeaderWithBackButton
+                            title={currentTaxRate?.name}
+                            threeDotsMenuItems={threeDotsMenuItems}
+                            shouldShowThreeDotsButton={!!canEdit}
+                            threeDotsAnchorPosition={styles.threeDotsPopoverOffsetNoCloseButton(windowWidth)}
+                        />
+                        <OfflineWithFeedback
+                            errors={ErrorUtils.getLatestErrorField(currentTaxRate, 'isDisabled')}
+                            pendingAction={currentTaxRate?.pendingFields?.isDisabled}
+                            errorRowStyles={styles.mh5}
+                            onClose={() => clearTaxRateFieldError(policyID, taxID, 'isDisabled')}
+                        >
+                            <View style={[styles.mt2, styles.mh5]}>
+                                <View style={[styles.flexRow, styles.mb5, styles.mr2, styles.alignItemsCenter, styles.justifyContentBetween]}>
+                                    <Text>{translate('workspace.taxes.actions.enable')}</Text>
+                                    <Switch
+                                        isOn={!currentTaxRate?.isDisabled}
+                                        accessibilityLabel={translate('workspace.taxes.actions.enable')}
+                                        onToggle={toggleTaxRate}
+                                        disabled={!canEdit}
+                                    />
+                                </View>
+                            </View>
+                        </OfflineWithFeedback>
+                        <OfflineWithFeedback
+                            errors={ErrorUtils.getLatestErrorField(currentTaxRate, 'name')}
```

Comment:
Why do we need to get latest error here?

Reply:
We want to display an error near the edited field which failed. I'm not sure I understand. Should we display the earliest error? The `src/pages/settings/Profile/Contacts/ContactMethodDetailsPage.tsx` has similar approach.